frontend/app/stores/banner.tsx

```diff
@@ -45,6 +48,7 @@ export function useBannerProfile(options?: {
 }
 
 export const banner = createBannerStore()
+PROFILE_IDS.forEach((id) => subscribeProfileToStorage(id))
```

Comment:
This statement will be executed as soon as this file is imported, including in tests. I'd suggest moving it to route file only.